src/meta/src/stream/stream_graph/fragment.rs

```diff
@@ -128,6 +128,10 @@ impl BuildingFragment {
                 table.database_id = job.database_id();
                 table.schema_id = job.schema_id();
                 table.fragment_id = fragment_id;
+                #[cfg(not(debug_assertions))]
```

Comment:
Where did we do this before? 👀

Reply:
We did it before for `mview_definition` in `StreamActor` before. 🥵